nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1246,6 +1246,7 @@ class SCORE_TYPE(str, Enum):
     VECTOR = "VECTOR"
     BM25 = "BM25"
     BOTH = "BOTH"
+    EXTERNAL = "EXTERNAL"
```

Comment:
Should we give more detail or "hiding" any implementation is fine?

Reply:
Actually, this should still be "vector" I think